debian/copyright

```diff
@@ -0,0 +1,51 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: sosreport
+Upstream-Contact: Bryn M. Reeves <bmr@redhat.com>
+Source: https://github.com/sosreport/sos
+
+Files: *
+Copyright: 2012-2013 Bryn M. Reeves
+	   2007-2013 Red Hat, Inc.
+License: GPL-2+
+
+Files: sos/report/plugins/kernelrt.py
+Copyright: 2012 Red Hat, Inc.
+License: GPL-2
+
+Files: debian/*
+Copyright: 2013 Adam Stokes <adam.stokes@ubuntu.com>
+License: GPL-2+
```

Comment:
Hmm.

The copyright assignment is generally done on a per-file basis within sos, and I don't want to leave a space for confusion here. I know, I know, GPL and all that, but this is one of those areas I have the mindset to be overly pedantic.

Is there a reason for specifically calling out `kernelrt.py` here?

Reply:
Good points, we just grabbed the stuff that was coming from the original packaging, and hasn't been reviewed for a while.

I have updated these now, to be more generic